src/terminal/parser/telemetry.cpp

```diff
@@ -34,8 +41,12 @@ TermTelemetry::TermTelemetry() :
 
 TermTelemetry::~TermTelemetry()
 {
-    WriteFinalTraceLog();
-    TraceLoggingUnregister(g_hConsoleVirtTermParserEventTraceProvider);
+    try

```

Comment:
this could be a function level try

Reply:
No, it can't. The compiler won't take it on a destructor, for whatever reason.